components/sysutils/net-snmp/history

```diff
@@ -1 +1,2 @@
-system/management/snmp/net-snmp/documentation@5.7.3-2018.0.0.4 system/management/snmp/net-snmp/demo
+library/python-2/net-snmp-26@5.4.4,5.11-2016.0.1.0
```

Comment:
appropriate net-snmp/documentation => demo rename SHOULD also be present, otherwise automatic updates will not be possible. Appropriate here means that system/management/snmp/net-snmp/documentation version should be > current one (for example, system/management/snmp/net-snmp/documentation@5.7.3-2018.0.0.5)

Reply:
The man files are now included in net-snmp-base.p5m.
So history should contain
`system/management/snmp/net-snmp/documentation@5.7.3-2018.0.0.5 system/management/snmp/net-snmp/demo`
which means uninstall older packages containing a mix of man and demo files and use the new demo package containing only demo files.